src/azul/__init__.py

```diff
@@ -91,7 +91,7 @@ def cache_per_thread(f, /):
 #: A type alias for annotating the return value of methods that return a
 #: ``furl`` instance that can be modified without side effects in the object
 #: whose method returned it.
-#
+#:
```

Comment:
s/Make Sphinx-style comments consistent in init file/Make Sphinx-style comments consistent/